Homework1/ParMatrixMult/ParMatrixMult/Matrix.cs

```diff
@@ -0,0 +1,135 @@
+// <copyright file="Matrix.cs" company="_">
+// Marina Popova, 2025, under MIT License.
+// </copyright>
+
+namespace ParMatrixMult;
+
+/// <summary>
+/// Class matrices.
+/// </summary>
+public class Matrix
+{
+    private int[,] matrix;
+    private int rows;
+    private int columns;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Matrix"/> class.
+    /// </summary>
+    /// <param name="rows">Number of rows.</param>
+    /// <param name="columns">Number of columns.</param>
+    public Matrix(int rows, int columns)
+    {
+        this.matrix = new int[rows, columns];
+        this.rows = this.matrix.GetLength(0);
+        this.columns = this.matrix.GetLength(1);
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Matrix"/> class.
+    /// </summary>
+    public Matrix()
+    {
+    }
+
+    /// <summary>
+    /// Returns a matrix from a file.
+    /// </summary>
+    /// <param name="filename">Name of file.</param>
+    /// <exception cref="ArgumentException">If filename does not include characters.</exception>
+    /// <exception cref="FileNotFoundException">If file doesn`t exist.</exception>
+    public void GetMatrixFromFile(string filename)
+    {
+        if (string.IsNullOrEmpty(filename))
+        {
+            throw new ArgumentException();
```

Comment:
```suggestion
            throw new ArgumentException(nameof(filename));
```
Чтобы сразу по сообщению об ошибке было понятно, что не так. 

Кстати, есть ArgumentException.ThrowIfNullOrEmpty